src/agent/kata-agent.service.in

```diff
@@ -20,3 +20,5 @@ LimitNOFILE=infinity
 # the runtime handles shutting down the VM.
 ExecStop=/bin/sync ; /usr/bin/systemctl --force poweroff
 FailureAction=poweroff
+# Discourage OOM-killer from touching the agent
+OOMScoreAdjust=-997
```

Comment:
Might be worth documenting in the commit why this value was chosen rather than `-1000`, presumably to make it highly unlikely the agent will be killed, but still allow for that possibility of the agent being justifiably killed if it suddenty decided to consume all available memory?